src/agent/debuglet.ts

```diff
@@ -392,7 +392,7 @@ export class Debuglet extends EventEmitter {
     //       named `response`) is not used.
     metadata.project(
         'project-id',
-        function(err: Error, _: http.ServerResponse, metadataProject: string) {
+        function(err: Error, _res: http.ServerResponse, metadataProject: string) {
```

Comment:
There are two distinct uses of underscores.. in parameter names, leading underscores indicate to the typescript compiler that the property may be unused. For property names, they primarily indicate to JS users of the library that the property is intended to be private.